packages/harpguru-core/src/components/option/option.tsx

```diff
@@ -44,6 +46,17 @@ export const Option = (props: OptionProps): React.ReactElement => {
     setTranslationY(nativeEvent.translationY)
   }
 
+  const previousOptionId = usePrevious(optionId, '')
+  const isUpdated = optionId !== previousOptionId
+  const optionUpdatedVal = useTimingTransition(isUpdated, {
+    duration: 300,
+    easing: Easing.inOut(Easing.circle),
+  })
+  const optionUpdateTransition = interpolate(optionUpdatedVal, {
```

Comment:
Realised that the naming of these animation values throughout the app isn't completely consistent. This one is Transition, but others are Translation. Transition kind of makes more sense since it's related to a transition value, but also it's not the transition value itself. That's `optionUpdatedVal` in this case, and this is and interpolation from that.

Reply:
Raised as tech debt: https://github.com/js-jslog/harpguru/issues/12